doc/source/contributing/sphinx_doc_build.md

```diff
@@ -1,15 +1,133 @@
 (sphinx_doc_build)=
 # Building the documentation
+Want to help with ArviZ documentation? There are several ways you could assist. You could help by suggesting changes via a pull request (PR), improving our documentation, reporting, and fixing issues. ArviZ documentation is constantly being improved and expanded by users and contributors, if you see anything that’s missing or have ideas on how to improve the existing documentation, please consider getting involved.
 
-The preferred workflow for contributing to ArviZ is to fork
-the [GitHub repository](https://github.com/arviz-devs/arviz/),
-clone it to your local machine, and develop on a feature branch. For a detailed description of the recommended development process, see the step-by-step guide {ref}`here <pr_steps>`.
+We recommend getting familiar with our [GitHub repository](https://github.com/arviz-devs/arviz/) before contributing to the ArviZ documentation.
+Our documentation is built using Sphinx and written in Markdown, but can also be written using reStructuredText or Jupyter notebook. The files are appended with the extensions .md, .rst, and .ipynb, respectively.
 
-If you are using Docker, see {ref}`building_doc_with_docker`.
+## Table of content:
+- [Getting started with documentation](Getting_started_with_documentation)
+- [Using Git on the command-line](Using_Git_on_the_command-line)
+- [Editing documents using Git](Editing_documents_using_Git)
+- [Building documents with Docker](Building_documents_with_Docker)
+- [Pull request checks](Pull_request_checks)
+- [Previewing doc changes](Previewing_doc_changes)
+- [Resources](Resources)
+   - Markdown guide
+   - reStructuredText guide
+   
+
+## Getting started with documentation
+
+Creating or improving the documentation requires that you build it locally. There are a few methods to choose from:
+
+- Using Git on the command line
+- Building documentation with Docker
```

Comment:
I find this a bit confusing. Git and GitHub are needed to contribute to ArviZ, independently of contributing documentation or not. There are 3 ways to build the changes in ArviZ documentation website, they all involve using sphinx in some way. Note that this only covers building the documentation which is a small step of the contributing process.

1. (preferred way) Build the documentation locally, either with `sphinx-build`, `sphinx-autobuild` or using either of those under the hood with an IDE specific extension that helps build and explore the generated files.
2. Build the documentation with Docker (technically also locally but with important differences)
3. Open a PR and look at the preview generated by readthedocs

When working locally or with Docker a 2nd step is needed to open/preview the generated html files in addition to generating them with sphinx.

Reply:
I'll make the necessary changes. I assumed the section on 'Building the documentation' could incorporate the information I added to make it easier for new contributors to contribute to the docs.